.github/workflows/lint-test.yml

```diff
@@ -1,30 +1,30 @@
-name: Build and Test
+name: Lint and Test
 
 on:
   push:
     branches: [ main ]
   pull_request:
 
 jobs:
-  build-test:
+  lint-test:
     name: Lint and Test
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [12.x, 14.x, 16.x]
     steps:
     - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: yarn --frozen-lockfile --ignore-scripts
```

Comment:
Why remove `--frozen-lockfile`? :thinking:

Reply:
This was a mistake, it has been corrected to `yarn --frozen-lockfile`